frontend/src/components/FilterTab.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./styles/FilterTab.scss";
+
+function FilterTab() {
+  return (
+    <div className="FilterTab">
+      <div className="center">
+        {/* GET CONCERNED TITLE TO INSERT IT THERE ( AMBASSADORS? CAUSES? ) */}
+        <p>Filtrer parmis les /* import name props */:</p>
+        <nav className="FiltersNav">
+          <ul>
+            {/* USE .map TO GENERATE BUTTONS ACCORDDING TO THE NUMBER OF CATEGORIES INVOLVED*/}
```

Comment:
Please don't include "TODO" instructions: either you do it, or you don't, but don't "hint" like that

Reply:
Done !